privacyidea/webui/login.py

```diff
@@ -62,6 +62,7 @@ def single_page_application():
 
     # The default theme. We can change this later
     theme = current_app.config.get("PI_CSS", DEFAULT_THEME)
+    theme = theme.strip('/');
```

Comment:
A bit nitpicky, but could you remove the ``;``? :)

Reply:
That's totally not nitpicking. That shouldn't have been there and has been exorcized.